packages/kokkos-kernels/src/sparse/KokkosSparse_gauss_seidel_handle.hpp

```diff
@@ -50,292 +50,292 @@
 
 namespace KokkosSparse{
 
-enum GSAlgorithm{GS_DEFAULT, GS_PERMUTED, GS_TEAM};
+  enum GSAlgorithm{GS_DEFAULT, GS_PERMUTED, GS_TEAM};
 
-template <class size_type_, class lno_t_, class scalar_t_,
-          class ExecutionSpace,
-          class TemporaryMemorySpace,
-          class PersistentMemorySpace>
-class GaussSeidelHandle{
-public:
-  typedef ExecutionSpace HandleExecSpace;
-  typedef TemporaryMemorySpace HandleTempMemorySpace;
-  typedef PersistentMemorySpace HandlePersistentMemorySpace;
+  template <class size_type_, class lno_t_, class scalar_t_,
```

Comment:
Changing the formatting without changing the content makes reviewing code hard, especially in this case, where you've made a subtle change to the algorithm that needs proofreading.